examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control/main.py

```diff
@@ -38,7 +38,7 @@ def validate(
 ) -> Tuple[Dict, int, int]:
     validator.seen = 0
     validator.jdict = []
-    validator.stats = []
+    validator.stats = dict(tp_m=[], tp=[], conf=[], pred_cls=[], target_cls=[])
```

Comment:
This code is not compatible with Python 3.8, which supports NNCF. Please rework it.